lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -22,9 +22,11 @@ class BarChartData extends AxisChartData {
     ),
     FlBorderData borderData,
     double maxY,
+    Color backgroundColor,
```

Comment:
Set a default color of Colors.transparent everywhere that you have this property ( I will mention them)